prow/gerrit/README.md

```diff
@@ -45,19 +45,19 @@ If you need extra features, feel free to introduce new gerrit API functions to t
 #### Adapter
 
 The adapter package implements a controller that is periodically polling gerrit, and triggering
-presubmit jobs base on your prow config.
+presubmit and postsubmit jobs based on your prow config.
 
 
 #### Reporter
 
-The reporter package handles send presubmit results back to gerrit. It implements a reporter interface
+The reporter package sends job results back to gerrit. It implements a reporter interface
 from [crier].
 
-The reporter will send an aggregated summary message, after all presubmits on a patchset finishes.
+The reporter will send an aggregated summary message, after all jobs on a patchset finishes.
```

Comment:
reporter will report twice, once for all presubmit jobs, and once for all postsubmit jobs :-)